composer.json

```diff
@@ -2,15 +2,27 @@
     "name": "tawk/tawk-wordpress",
     "description": "Wordpress plugin for tawk.to",
     "type": "project",
-    "license": "OSL-3",
-    "version": "0.6.0",
+    "license": "OSL-3.0",
+    "version": "0.7.0",
+    "require": {
+        "tawk/url-utils": "dev-feature/url-and-path-matching-modules"
+    },
     "require-dev": {
         "squizlabs/php_codesniffer": "^3.6",
         "wp-coding-standards/wpcs": "^2.3",
         "phpcompatibility/php-compatibility": "^9.3",
         "woocommerce/woocommerce-sniffs": "^0.1.1"
     },
+    "repositories": {
+        "tawk-url-utils": {
+            "type": "vcs",
+            "url": "https://github.com/alvinjohnsonso/tawk-url-utils.git"
```

Comment:
Will be updated to https://github.com/tawk/tawk-url-utils.git along with the version once the update's been released